tests/test-team/setup/session.setup.ts

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable no-await-in-loop */
+/* eslint-disable unicorn/prefer-module */
+/* eslint-disable import/no-unresolved */
+/* eslint-disable @typescript-eslint/no-var-requires */
+
+import { test as setup } from '@playwright/test';
+import { generateClient } from 'aws-amplify/data';
+import { Amplify } from 'aws-amplify';
+import { TemplateMgmtChoosePage } from '../pages/template-mgmt-choose-page';
+import { Session } from '../../../src/utils/types';
+import { Schema } from '../../../amplify/data/resource';
+import { TemplateMgmtCreatePage } from '../pages/template-mgmt-create-page';
+
+const config = require('@/amplify_outputs.json');
+
+setup('setup session data', async () => {
+  const sessionData: Session[] = [
+    ...TemplateMgmtChoosePage.sessionData,
+    ...TemplateMgmtCreatePage.sessionData,
+  ];
+
+  Amplify.configure(config);
+
+  const client = generateClient<Schema>({ authMode: 'iam' });
+
+  for (const data of sessionData) {
+    await client.models.SessionStorage.create(data);
```

Comment:
The benefit of this is that it is quick to develop now but the downside is that the test is not independent of the application code because it is using the graphQL API to create the data.  An alternative would be to go directly to the database via an independent auth role.  I'm not sure how this approach will work when we have a more mature auth layer as we won't have guest access to the APIs.

Reply:
Updated so as not to use the GraphQL api to create data ... the dynamo table name is ascertained by matching GraphQL api endpoints between that in the amplify outputs and that listed using the a method from AppSync SDK. Then uploaded directly to dynamo.